homeassistant/components/websocket_api/commands.py

```diff
@@ -384,9 +385,8 @@ def _send_handle_entities_init_response(
     connection: ActiveConnection, msg_id: int, serialized_states: list[str]
 ) -> None:
     """Send handle entities init response."""
-    joined_states = ",".join(serialized_states)
     connection.send_message(
-        construct_event_message(msg_id, f'{{"a":{{{joined_states}}}}}')
+        f'{{"id":{msg_id},"type":"event","event":{{"a":{{{",".join(serialized_states)}}}}}}}'
```

Comment:
Its a bit unfortunately that the escape sequence for a `{` is `{{`. This is a bit hard to read, but its is correct and well tested

Reply:
Have you seen this? https://github.com/ijl/orjson#fragment seems it finally got added 2 months ago.

Reply:
I forgot about that. I'll give it a shot. I spent most of the day in aiohttp so I didn't get a chance to come back to this

Reply:
Could probably help here as well https://github.com/home-assistant/core/blob/8e112c04fb5454be64efe9de12b2ceb34f00cc8a/homeassistant/components/websocket_api/http.py#L163